<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QBCfgTabPageUsersUi</class>
 <widget class="QWidget" name="QBCfgTabPageUsersUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Users</string>
  </property>
  <layout class="QHBoxLayout">
   <item>
    <widget class="Q3GroupBox" name="groupBox4">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QBUserListView" name="userList" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QPushButton" name="userNewButton">
          <property name="text">
           <string>New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="userEditButton">
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="userDeleteButton">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>31</width>
            <height>71</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QBUserListView</class>
   <extends>QWidget</extends>
   <header location="global">q4banking/qbuserlist.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
